src/magic.cpp

```diff
@@ -260,6 +298,10 @@ void spell_type::check_consistency()
             debugmsg( string_format( "ERROR: %s has positive increment and base_casting_time > final_casting_time",
                                      sp_t.id.c_str() ) );
         }
+        std::set<spell_id> spell_effect_list;
+        if( spell_infinite_loop_check( spell_effect_list, sp_t.id ) ) {
+            debugmsg( string_format( "ERROR: %s has infinite loop in extra_effects", sp_t.id.c_str() ) );
```

Comment:
```suggestion
            debugmsg( "ERROR: %s has infinite loop in extra_effects", sp_t.id.c_str() );
```
`debugmsg` calls `string_format` anyway.

Applies as well to other places where `debugmsg` and `string_format` are used together.